src/content/docs/en/reference/publish-to-npm.mdx

```diff
@@ -17,7 +17,7 @@ Check out [Astro Community's component template](https://github.com/astro-commun
 
 ## Quick Start
 
-To get started developing your component quickly, we have a template already setup for you.
+To get started developing your component quickly, you can use a template already setup for you.
```

Comment:
```suggestion
To get started developing your component quickly, you can use a template already setup for you.
```
I do like this better without the we, but as mentioned in another comment, this is not a case of "wrong we" usage.